tensorflow/docs_src/programmers_guide/debugger.md

```diff
@@ -9,10 +9,16 @@ lets you view the internal structure and states of running TensorFlow graphs
 during training and inference, which is difficult to debug with general-purpose
 debuggers such as Python's `pdb` due to TensorFlow's computation-graph paradigm.
 
-> NOTE: The system requirements of tfdbg on supported external platforms include
-> the following. On Mac OS X, the `ncurses` library is required. It can be
-> installed with `brew install homebrew/dupes/ncurses`. On Windows, `pyreadline`
-> is required. If you use Anaconda3, you can install it with a command
+> NOTE: TensorFlow debugger uses a
+> [curses](https://en.wikipedia.org/wiki/Curses_\(programming_library\))-based
+> text user interface. On Mac OS X, the `ncurses` library is required and can
+> be installed with `brew install homebrew/dupes/ncurses`. On Windows, an
+> unofficial curses package for various Python versions can be downloaded
```

Comment:
I personally find that distribution of curses on Windows not 100% reliable. For example, many font attributes, such as background colors and underscores, do not get rendered correctly on some machines. So while I'm okay with adding mention of that unofficial distribution of curses, I'd suggest moving it after the Window pyreadilne option.

Reply:
Also, please add warning to the user that the Windows distribution may not work as reliably as curses on Linux and Mac.